README.md

```diff
@@ -1,44 +1,83 @@
 # GameOn Backend Api
-The Backend app for GameOn is hosted on Heroku, written in Python, using the Flask framework with data accessed through MongoDB.
-__GameOnBE: 'https://game-on-pro.herokuapp.com'__
+Host:       Heroku 
+Language:   Python
+Framework:  Flask 
+Database:   SQLAlchemy
+**GameOnBE: 'https://game-on-pro.herokuapp.com'**
 
 ### Formatting
-The GameOnBE App follows REST API convention, displaying data in JavaScript Object Notation(JSON).
+- The GameOnBE App follows REST API convention, displaying data in JavaScript Object Notation(JSON).
+- Each collection is returned within a 'data:' hash
+- Example:
+_request_: GET 'http://game-on-pro.herokuapp.com/api/v1/events'
+_response body_: { 
+  "data": [
+    {
+      "activity": "Soccer",
+      "attending": true,
+      "current_participant_count": 2,
+      "date": "2020-04-15",
+      "description": "",
+      "duration": "30 min",
+      "equipment": "ball",
+      "id": 49,
+      "lat_long": "39.7365497,-104.9899995",
+      "location": "Civic Center Park",
+      "max_participant_count": 2,
+      "skill_level": "Just for fun!",
+      "start_time": "6:00AM" 
+    }
+  ]
+}
 
 ### Getting Started
-We currently have data available for activities (index, show), 10 instances pre-populated within our Database (MongoDB). 
-
-### Future Additions
-- Events (Index, Show, Create, Destroy)
-- Feature: Activities can be added when creating New Events.
+- To consume GameOnBE data, we will send requests to: 'https://game-on-pro.herokuapp.com/api/v1/events'
+- Instructions for CRUD endpoints include: [GET, POST, PATCH, DELETE]
 
 ### Endpoints
```

Comment:
Can we add something here with like "base URL for all endpoints: 'http://game-on-pro.herokuapp.com'"